src/convertToObject.js

```diff
@@ -11,8 +11,24 @@
  *
  * @return {object}
  */
+/* eslint-enable */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
> According to the checklist, you should give this variable a different name:
> 
> 2. [CODE STYLE] - Avoid variable namings like `array` , `object` `arr`, `obj`, `result`. It will decrease code readability

Good point, naming is important